packages/components/pie-webc-core/src/functions/customEventDispatcher.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Dispatch a custom event.
+ *
+ * To be used whenever we have behavioural events we want to
+ * bubble up through the component.
+ * @param {Element} element The element to dispatch the event from.
+ * @param {string} eventType
+ * @param {any} detail
+ */
+export function dispatchCustomEvent (element: Element, eventType: string, detail?: CustomEventInit['detail']) : void {
```

Comment:
we should add some unit tests for this function

Reply:
I added a browser test taking formControlMixin.browser.spec.ts as an example but looks like while it passes as part of browser tests in fails as part of unit tests. I am not sure what I did wrong, should I exclude it from unit tests somehow?

Reply:
All fixed, thanks for your help @jamieomaguire 
For the ci tests to pass I had to leave link to the root vite config in webc-core package.json but updated it to include not only mixin folder